jobs/consul/templates/consul/agent.json.erb

```diff
@@ -20,7 +21,6 @@
     bind_addr: '0.0.0.0',
     client_addr: client_addr,
     advertise_addr: my_ip,
-    domain: 'consul',
```

Comment:
Why removing this? If it's to allow override by agent_config then perhaps setup the override in the variable above?